spec/models/finance/schedule/npq_specialist_spec.rb

```diff
@@ -10,7 +10,7 @@
   it "seeds from csv" do
     schedule = described_class.find_by(schedule_identifier: "npq-specialist-spring")
     expect(schedule).to be_present
-    expect(schedule.milestones.count).to eql(3)
+    expect(schedule.milestones.count).to eql(4)
```

Comment:
Checking the csv, this value should be 3 milestones rather than 4, worth finding out how we're getting the fourth?

Reply:
Pretty sure this is because the CSV is adding to the already set up cohort